desktop/src/com/mygdx/game/client/input/TechnologyInputAdapter.java

```diff
@@ -0,0 +1,30 @@
+package com.mygdx.game.client.input;
+
+import com.badlogic.gdx.Input;
+import com.badlogic.gdx.InputAdapter;
+import com.mygdx.game.client.GdxGame;
+import lombok.NonNull;
+import lombok.extern.java.Log;
+
+@Log
+public class TechnologyInputAdapter extends InputAdapter {
+
+  private final GdxGame game;
+
+  public TechnologyInputAdapter(@NonNull GdxGame game) {
```

Comment:
TechnologyTreeScreenInput czy coś, bo w sumie nie wiem od czego to jest input po nazwie